src/Framework/TaskStartedEventArgs.cs

```diff
@@ -107,6 +157,7 @@ internal override void CreateFromStream(BinaryReader reader, int version)
             taskFile = reader.ReadByte() == 0 ? null : reader.ReadString();
             LineNumber = reader.Read7BitEncodedInt();
             ColumnNumber = reader.Read7BitEncodedInt();
+            TaskAssemblyName = reader.ReadByte() == 0 ? null : reader.ReadString();
```

Comment:
```suggestion
            TaskAssemblyName = reader.ReadOptionalString();
```

Reply:
Thanks, I changed also other instances in this method.